alias.ts

```diff
@@ -3,29 +3,35 @@ import { resolve } from 'path'
 const r = (p: string) => resolve(__dirname, p)
 
 export const alias: Record<string, string> = {
-  '@unocss/shared-integration': r('./packages/shared-integration/src/'),
-  '@unocss/shared-docs': r('./packages/shared-docs/src/'),
-  '@unocss/shared-common': r('./packages/shared-common/src/'),
+  '@unocss/astro': r('./packages/astro/src/'),
   '@unocss/autocomplete': r('./packages/autocomplete/src/'),
   '@unocss/cli': r('./packages/cli/src/'),
-  '@unocss/core': r('./packages/core/src/'),
   '@unocss/config': r('./packages/config/src/'),
+  '@unocss/core': r('./packages/core/src/'),
+  '@unocss/extractor-pug': r('./packages/extractor-pug/src/'),
   '@unocss/inspector': r('./packages/inspector/node/'),
+  '@unocss/nuxt': r('./packages/nuxt/src/'),
   '@unocss/preset-attributify': r('./packages/preset-attributify/src/'),
   '@unocss/preset-icons': r('./packages/preset-icons/src/'),
   '@unocss/preset-mini': r('./packages/preset-mini/src/'),
+  '@unocss/preset-rem-to-px': r('./packages/preset-rem-to-px/src/'),
   '@unocss/preset-tagify': r('./packages/preset-tagify/src/'),
   '@unocss/preset-typography': r('./packages/preset-typography/src/'),
   '@unocss/preset-uno': r('./packages/preset-uno/src/'),
   '@unocss/preset-web-fonts': r('./packages/preset-web-fonts/src/'),
   '@unocss/preset-wind': r('./packages/preset-wind/src/'),
-  '@unocss/preset-rem-to-px': r('./packages/preset-rem-to-px/src/'),
+  '@unocss/reset': r('./packages/reset/scripts/'),
+  '@unocss/runtime': r('./packages/runtime/src/'),
+  '@unocss/scope': r('./packages/scope/'),
+  '@unocss/shared-common': r('./packages/shared-common/src/'),
+  '@unocss/shared-docs': r('./packages/shared-docs/src/'),
+  '@unocss/shared-integration': r('./packages/shared-integration/src/'),
+  '@unocss/transformer-attributify-jsx': r('./packages/transformer-attributify-jsx/src/'),
+  '@unocss/transformer-compile-class': r('./packages/transformer-compile-class/src/'),
   '@unocss/transformer-directives': r('./packages/transformer-directives/src/'),
   '@unocss/transformer-variant-group': r('./packages/transformer-variant-group/src/'),
-  '@unocss/transformer-compile-class': r('./packages/transformer-compile-class/src/'),
-  '@unocss/transformer-attributify-jsx': r('./packages/transformer-attributify-jsx/src/'),
+  'unocss': r('./packages/unocss/src/'),
```

Comment:
Why don't we just sort alphabetically?

Reply:
🐻‍❄️ I think it's easy for contributors to know where the alias should go
![image](https://user-images.githubusercontent.com/76603360/187169266-9abc04b3-3d3f-4d03-ad96-6231d1075190.png)

Reply:
> 

Before that, I'd also like to implement a Node script that implements it 😂, If you want to do that, call me. I would like to do it